src/components/App.tsx

```diff
@@ -13,7 +13,7 @@ import { getTheme } from 'theme';
 
 import { AppContextProvider } from 'context/AppContext';
 import { UserContextProvider, UserContext } from 'context/UserContext';
-import { useUnauthorizedApi } from 'hooks/common/useDataApi';
+// import { useUnauthorizedApi } from 'hooks/common/useDataApi';
```

Comment:
Do we need to keep the comments about unauthorizedApi? If the plan is to use them again then it's fine otherwise you can probably remove them.

Reply:
We may, or not, not sure yet, I plan to clean it up after it is solved if it's fine by you.